runtime/src/tasks/massivethreads/tasks-massivethreads.c

```diff
@@ -66,17 +61,19 @@ static int tasking_layer_active = 0;
 static int worker_in_cs_beforeinit = 0;
 static thread_local_data* s_tld;
 
-static const chpl_task_prvDataImpl_t s_def_chpl_data=
+// task-private data of comm task
```

Comment:
My recollection is that tasks-massivethreads.c already didn't work, due to other tasking interface changes not having been done.  Does it build and test out for you?